pkg/rancher-desktop/main/mainEvents.ts

```diff
@@ -100,6 +100,12 @@ interface MainEventNames {
    */
   'diagnostics-trigger'(id: string): DiagnosticsCheckerResult | undefined;
 
+  /**
+   * Emitted when an extension is uninstalled via the extension manager.
+   * @param id The ID of the extension that was uninstalled.
+   */
+  'extensions/ui/uninstall'(id: string): void;
```

Comment:
I attempted to keep this in line with other ipc events from `electron-ipc.d.ts`, but I'm open to renaming if we think there's something better.